src/goTest.ts

```diff
@@ -112,6 +140,22 @@ export function testCurrentFile(timeout: string) {
 }
 
 /**
+ * Kill all running go task.
+ *
+ */
+export function killRunning() {
```

Comment:
@ironcladlou 
It is only track PIDs which started by Go:Test, indeed not safed.
but ,to track running test which is started by vscode-go plugin is hard for VScode, i think. 

finally, this features to kill running process is very useful when coding by TDD, it will pull request again.